src/enzyme_ad/jax/primitives.py

```diff
@@ -1050,6 +1074,9 @@ def fwd_partial_eval(trace, *args, **kwargs):
 
 
 def primal_partial_eval(trace, *args, **kwargs):
+    print("trace ", trace)
+    print("args", args)
+    print("kwargs", kwargs)
```

Comment:
Ditto. We can use `import logging` and potentially even attach to the jax logger if desired.